src/tm/class-wpml-tm-page-builders-hooks.php

```diff
@@ -57,7 +57,13 @@ public function pro_translation_completed_action( $new_post_id, array $fields, s
 	public function adjust_translation_fields_filter( array $fields, $job ) {
 		$worker = $this->get_worker();
 
-		return $worker->adjust_translation_fields_filter( $fields, $job );
+		$fields = $worker->adjust_translation_fields_filter( $fields, $job );
+		foreach ( $fields as &$field ) {
```

Comment:
@kagg-design, this logic to add the wrap tag to the title should go inside `$worker->adjust_translation_fields_filter` IMO. There you will already find a loop where you can do this.

As you can see, the class `WPML_TM_Page_Builders_Hooks` is just hooking actions/filters but delegates the logic to a worker.

Reply:
@strategio agreed, done. Thanks for comment.